### DISCUSSION TITLE
Quickfix in nvchad?

### DISCUSSION THREAD

**Discussion by Enzodtz** [NvChad/NvChad#2553] [Q&A] [answered]
Hello, I'm new to nvim and come from vscode. 

I was wondering if there's any feature like [quickfix](https://code.visualstudio.com/docs/csharp/refactoring#:~:text=Visual%20Studio%20Code%20gives%20you,Quick%20Fixes%20and%20refactoring%20options.) in which you can open a menu that will show some possible solutions for a warning or error that is happening.

Example:

```c
int main(int argc, char *argv[]) {
  printf('Hello, world'); // printf is not defined

  return 0;
}
```

Quickfix would say something like "add `#include <stdio.h>`.

I know there's a quickfix feature in nvim but it is not like that so I wanted to see if there's anything like that. Thank you.

**Comment by siduck** [comment, marked as answer]
those are called code actions i think, check default mapping list

**Reply by Enzodtz** [reply]
Awesome, thank you!
